lib/matplotlib/axes/_base.py

```diff
@@ -214,6 +214,7 @@ class _process_plot_var_args:
     """
 
     def __init__(self, command='plot'):
+        _api.check_in_list(['plot', 'fill', 'mirror'], command=command)
```

Comment:
We wanted to be able to re-use all of the machinery for processing the vargs, but get the per-artist args/wkwargs out so the we can create new-style artist from them (which we can not inject into core Matplotlib yet).

I agree it is not great, but was going for the minimal change that would unblock the data prototype work without having to re-write this whole API!

Instead of `"mirror"` use `"no_artist"` instead?

Reply:
This is all internal, so renaming would be possible. However there's a slight complication that `command` is used in `raise _api.kwarg_error(self.command, pos_only)`. With that we inject the usage context name into the class, so that when called from `plot()` it can issue an error containing "plot" and likewise for "fill". This means that `_process_plot_var_args` is not really context free and you'd have to hard-code in which places you want to use it to support a proper exception.

The "cheap" solution would be `command="UNKNOWN"` or similar, which would result in

> UNKNOWN() got an unexpected keyword argument 'x'

That's sort of ok.

The IMHO slightly better would be to decouple the function from the caller, i.e. return a generic "unexpected keyword arguments 'x'" and - if desired - catch and ammend that in the caller. This would give the freedom to rename `command` - e.g. to `artist` with values Line2D, Polygon and None.